iis-wf/iis-wf-import/src/main/resources/eu/dnetlib/iis/wf/importer/content_url/chain/oozie_app/workflow.xml

```diff
@@ -146,9 +161,16 @@
 	        <arg>org.apache.avro.Schema.Type.STRING</arg>
 	        <capture-output />
 	    </java>
-	    <ok to="objectstore-content-url-importer" />
```

Comment:
This particular change could be confusing but it is bound to a [specific commit](https://github.com/openaire/iis/pull/1361/commits/2439e414f8b591fc81df65a952b2813c726fa5c8) which is closing the #1342 issue spotted when working on #1298. 

Maybe it is not worth to elevate it to a dedicated PR but we should find a nice way to indicate this solution somehow in the commit comment once the commits get squshed before the merge.